driver/image_adjust_brightness_driver.hpp

```diff
@@ -0,0 +1,301 @@
+ *
+ *******************************************************************************/
+
+#ifndef GUARD_MIOPEN_IMAGE_ADJUST_BRIGHTNESS_DRIVER_HPP
+#define GUARD_MIOPEN_IMAGE_ADJUST_BRIGHTNESS_DRIVER_HPP
+
+#include "../test/tensor_holder.hpp"
+#include "../test/verify.hpp"
+#include "InputFlags.hpp"
+#include "driver.hpp"
+#include "image_adjust_driver_common.hpp"
+#include "miopen/miopen.h"
+#include "miopen/tensor.hpp"
+#include "miopen/tensor_view.hpp"
+#include "tensor_driver.hpp"
+#include "tensor_view.hpp"
+#include "timer.hpp"
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+
+template <typename Tgpu, typename Tref>
+void mloImageAdjustBrightnessRunHost(const Tgpu* input,
+                                     Tref* output,
+                                     miopen::TensorDescriptor inputDesc,
+                                     miopen::TensorDescriptor outputDesc,
+                                     const float brightness_factor)
+{
+    tensor_view_4d_t input_tv  = get_inner_expanded_4d_tv(inputDesc);
+    tensor_view_4d_t output_tv = get_inner_expanded_4d_tv(outputDesc);
+
+    size_t N = inputDesc.GetElementSize();
+
+    for(size_t gid = 0; gid < N; gid++)
+    {
+        Tref pixel  = get4DValueAt(input, input_tv, gid);
+        Tref result = clamp(pixel * brightness_factor, Tref(0.0f), Tref(1.0f));
+        set4DValueAt(output, output_tv, gid, result);
+    }
+}
+
+template <typename Tgpu, typename Tref>
+class ImageAdjustBrightnessDriver : public Driver
+{
+public:
+    ImageAdjustBrightnessDriver() : Driver()
+    {
+        miopenCreateTensorDescriptor(&inputTensorDesc);
+        miopenCreateTensorDescriptor(&outputTensorDesc);
+
+        data_type = miopen_type<Tgpu>{};
+    }
+
+    int AddCmdLineArgs() override;
+    int ParseCmdLineArgs(int argc, char* argv[]) override;
+    InputFlags& GetInputFlags() override { return inflags; }
+
+    int GetandSetData() override;
+
+    int AllocateBuffersAndCopy() override;
+
+    int RunForwardGPU() override;
+    int RunForwardCPU();
+
+    int RunBackwardGPU() override;
+    int RunBackwardCPU();
+
+    int VerifyForward() override;
+    int VerifyBackward() override;
+
+    ~ImageAdjustBrightnessDriver() override
+    {
+        miopenDestroyTensorDescriptor(inputTensorDesc);
+        miopenDestroyTensorDescriptor(outputTensorDesc);
+    }
+
+private:
+    InputFlags inflags;
+
+    int forw;
+
+    miopenTensorDescriptor_t inputTensorDesc;
+    miopenTensorDescriptor_t outputTensorDesc;
+
+    std::unique_ptr<GPUMem> input_gpu;
+    std::unique_ptr<GPUMem> output_gpu;
+
+    std::vector<Tgpu> in_host;
+    std::vector<Tgpu> out_host;
+
+    std::vector<Tref> out_ref;
+
+    float brightness_factor;
+};
+
+template <typename Tgpu, typename Tref>
+int ImageAdjustBrightnessDriver<Tgpu, Tref>::AddCmdLineArgs()
+{
+    inflags.AddInputFlag("forw", 'F', "1", "Run only the forward pass", "int");
+
+    inflags.AddTensorFlag("input", 'I', "1x3x96x96", "Input Tensor Size");
```

Comment:
Please add an example to the description of this flag for convenient usage

Reply:
Since the flag is `AddTensorFlag`, they are added automatically by the framework

![image](https://github.com/moreh-dev/MIOpen/assets/76864037/cb43893a-1ae1-4a0e-82fa-394106266691)